dhall-csv/Setup.hs

```diff
@@ -0,0 +1,2 @@
+import Distribution.Simple
+main = defaultMain
```

Comment:
Please use newlines at the end of files. Otherwise missing newlines often result in noisy commit diffs.

Maybe you can configure this in your text editor?!

Reply:
Thanks, I've just configured my editor to add final newlines and remove trailing whitespaces on save